flink-table/flink-sql-gateway/src/test/java/org/apache/flink/table/gateway/service/operation/OperationManagerTest.java

```diff
@@ -129,29 +135,54 @@ public void testCancelOperation() throws Exception {
     }
 
     @Test
-    public void testCancelOperationByForce() throws Exception {
-        AtomicReference<Throwable> exception = new AtomicReference<>(null);
+    public void testCancelUninterruptedOperation() throws Exception {
```

Comment:
nit: modifier could be package private
```suggestion
    void testCancelUninterruptedOperation() throws Exception {
```